src/cil.mli

```diff
@@ -523,138 +523,140 @@ and varinfo = {
     (** Indicates whether the vdescr above is a pure expression or call.
      *  Printing a non-pure vdescr more than once may yield incorrect
      *  results. *)
+
+    mutable vdeclinstgenerated: bool;
```

Comment:
inst=instance? Better name or some comment.

Reply:
Renamed to vhasdeclinstruction